features/dd-sdk-android-ndk/src/androidTest/kotlin/com/datadog/android/ndk/NdkTests.kt

```diff
@@ -95,45 +93,65 @@ class NdkTests {
 
     @Test
     fun mustNotWriteAnyLog_whenHandlingSignal_whenConsentUpdatedToPending() {
-        val signal = forge.anInt(min = 1, max = 32)
-        // we need to keep this this size because we are using a buffer of [30] size in c++ for
-        // the error.signal attribute
-        val signalName = forge.anAlphabeticalString(size = 20)
-        val errorMessage = forge.anAlphabeticalString()
-        val errorStack = forge.anAlphabeticalString()
+        val fakeSignal = forge.aPositiveInt(true)
+        val fakeSignalName = forge.anAlphabeticalString(size = 20)
```

Comment:
> [!NOTE]
> Can you use a more random thing than 20, or at least use a constant explaining the 30 characters constraint

Reply:
We actually do not need these limitations anymore as this was needed before when that part of code was written in pure C and we used fixed length memory allocation for that. Currently that code is C++ and allows dynamic allocation.